components/SSI/DIDxWallet/Wallet/DIDOperations/SocialRecovery/index.tsx

```diff
@@ -263,6 +262,35 @@ function Component() {
     }
   };
 
+  const resolveDid = async (_username: string) => {
+    await fetchAddr({ net, _username, _domain: "did" })
+      .then(async () => {
+        toast.info(`${_username} is available`, {
```

Comment:
when successful, just continue - do not open toast (btw if the fetch address is successful, it returns an address, which means the username is not available but already assigned to an address)

Reply:
Copy on this

Reply:
Yes, I mean the username is available for social recovery. Anyway already remove the toast. Thanks!